pkg/app/bundle_event_handler.go

```diff
@@ -4,10 +4,12 @@ import (
 	"log"
 
 	"github.com/atlassian/smith"
+	"k8s.io/apimachinery/pkg/runtime"
```

Comment:
Style: there should be a blank line between imports from same project and 3rd party imports.